radixdlt/src/main/java/com/radixdlt/middleware2/network/TestEventCoordinatorNetwork.java

```diff
@@ -28,13 +28,12 @@
 import io.reactivex.rxjava3.subjects.PublishSubject;
 
 import java.util.Collections;
-import java.util.Deque;
```

Comment:
This class looks like it is in the wrong part of the source tree.  Should be in radixdlt/src/test/java

Reply:
I think this module is hefty enough and gonna need its own unit tests so `/src/test/java` doesn't seem appropriate. Thoughts?

Reply:
There's no problem with writing unit tests for unit tests :)

I think from a deliverable point of view, it would be better to exclude code from the runtime package that wasn't required for runtime.  Some important security benefits, including smaller code footprint, a no real drawbacks that I can see.

Reply:
I think these simulation tests need some re-thinking on where they belong so I'll just add another task for that.